src/test/agent/test_systemd.py

```diff
@@ -1,69 +1,85 @@
 import unittest
-from mock import MagicMock
+from mock import MagicMock, call
 from spacel.model import AgentManifest
 from spacel.agent.systemd import SystemdUnits
 
 INSTANCE_ID = 'i-123456'
 SYSTEMD_SERVICE = 'foo.service'
+TIMER_SERVICE = 'bar.service'
+TIMER_NAME = 'bar.timer'
 
 
 class TestSystemdUnits(unittest.TestCase):
     def setUp(self):
-        self.unit = MagicMock()
-        self.unit.properties.Id = SYSTEMD_SERVICE
+        self.unit1 = MagicMock()
+        self.unit1.properties.Id = SYSTEMD_SERVICE
+        self.unit2 = MagicMock()
```

Comment:
Naming: `timer_service` would make later uses more readable than remembering which unit{1,3} is which